### DISCUSSION TITLE
Does not have temperature target option

### DISCUSSION THREAD

**Discussion by Ramensoup420** [seerge/g-helper#3320] [General]
Hey guys, for some reason I don`t have temperature target option.
I have Asus Duo 15 se, RTX 3080 with Amd cpu.
Some one knows what to do? its really important to me to have this option.
![asus duo](https://github.com/user-attachments/assets/a3cfeba7-8beb-482a-8488-fa9fd7002d19)

**Comment by seerge** [comment]
@Ramensoup420 hello, if you never had GPU temperature target in Armoury - then it means your device does not support it, and you won't have it here.
